### PR TITLE
UPDATE: Stepper exports stateless version

### PR DESCRIPTION
**Pull request by filipdanisko**
Adding stateless export of Stepper.

adding props to statefull stepper
- titleIncrement
- titleDecrement
to handle translation of buttons.<br/><br/><br/><url>LiveURL: https://orbit-components-update-stepper-stateless.surge.sh</url>

### CODE REVIEW

**Comment by tomashapl** [comment]
Just IDEA: should be more nice to export it as InputStepper.Stateless ?

File: src/Stepper/index.js [hunk 1]
```diff
@@ -85,39 +46,37 @@ class Stepper extends React.PureComponent<Props, State> {
...
+        onDecrement={this.decrementCounter}
+        titleIncrement={titleIncrement}
+        titleDecrement={titleDecrement}
+      />
     );
   }
 }
 
 export default Stepper;
+export { default as InputStepperStateless } from "./StepperStateless";
```
Comment (tomashapl) [inline src/Stepper/index.js]: `StepperStateless` 🙏

File: src/utils/validateDecrement/index.js.flow [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+export type Params = {|
+  value: number,
+  minValue?: number,
+  step?: number,
+|};
+
+export type ValidateDecrement = Params => number;
+
+declare export default ValidateDecrement;
```
Comment (tomashapl) [inline src/utils/validateDecrement/index.js.flow]: This is the candidate for some unit tests 🙏

**Comment by vepor** [comment]
I would not do it as it's not probably tree shakable and we don't do it for other components, e.g. `List.ListItem`

File: src/Stepper/index.js.flow [hunk 3]
```diff
@@ -4,18 +4,23 @@
...
-export type Props = {|
+export type SharedProps = {|
   ...Globals,
-  +defaultValue?: number,
-  +step?: number,
   +name?: string,
   +disabled?: boolean,
   +maxValue?: number,
   +minValue?: number,
+  +titleIncrement?: () => string | string,
```
Comment (tomashapl) [inline src/Stepper/index.js.flow]: `string | (any => string)`

**Pull request merged**